src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,43 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    static {
+        System.out.println("Class IndexController was loaded");
+    }
+
```

Comment:
It was in the Bogdan's video, but it doensn't nessesaraly in this task.
It wrote this in my task but it mush be deleted.

Reply:
I was added this one to more informativity.